### DISCUSSION TITLE
Errors downloading model files

### DISCUSSION THREAD

**Discussion by sachel85** [Purfview/whisper-standalone-win#222] [Q&A]
It looks like I am getting an SSLError when I try to run the standalone whisper for the first time.  Any tricks to getting the model files downloaded?

```
An error occured while synchronizing the model Systran/faster-whisper-medium from the Hugging Face Hub:
(MaxRetryError("HTTPSConnectionPool(host='huggingface.co', port=443): Max retries exceeded with url: /api/models/Systran/faster-whisper-medium/revision/main (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:1131)')))"), '(Request ID: ec4faccf-3680-4d93-a522-c2ff02076c9d)')
Trying to load the model directly from the local cache, if it exists.
```

**Comment by Purfview** [comment]
Your error: `certificate verify failed: self signed certificate in certificate chain`
There is solution to similar - `certificate verify failed: unable to get local issuer certificate` error: https://github.com/Purfview/whisper-standalone-win/issues/75#issuecomment-1733592055

**Reply by sachel85** [reply]
Thank you! I was able to run the following:
```
set HTTPS_PROXY=huggingface.co:443
set HTTP_PROXY=huggingface.co:443
```
It seems like that eliminated the SSL error, but now I am getting this:

```
(ProtocolError('Connection aborted.', ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None)), '(Request ID: 01c76fa2-f7bd-4039-a183-df668af63240)')
Trying to load the model directly from the local cache, if it exists.
```

**Reply by Purfview** [reply]
huggingface is not some proxy...

Read there: https://stackoverflow.com/a/76602300

Btw, you can download models manually. [read info on the main page]
